jobs/build/custom/Jenkinsfile

```diff
@@ -17,7 +17,7 @@ node {
                     commonlib.oseVersionParam('BUILD_VERSION'),
                     [
                         name: 'VERSION',
-                        description: 'Version string for build without leading "v" (i.e. 4.0.0)',
+                        description: 'Version string for build (e.g. 4.0.0)',
```

Comment:
I liked that note. Because I did it too many times :P

Reply:
The change here is that it works with or without the "v" so you don't have to remember